src/electron/platform/android/setup/steps/configuring-port-forwarding.ts

```diff
@@ -4,7 +4,11 @@
 import { AndroidSetupStepConfig } from 'electron/platform/android/setup/android-setup-steps-configs';
 
 export const configuringPortForwarding: AndroidSetupStepConfig = deps => ({
-    actions: {},
+    actions: {
+        cancel: () => {
```

Comment:
Cancel button is intentionally disabled in this step because we have no way to cancel the underlying. There's no harm in having the plumbing here, but it will never be exercised